app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt

```diff
@@ -128,21 +128,24 @@ class MeetingRequestService : FirebaseMessagingService() {
         val stringReason =
             when (message) {
               "DISTANCE" -> DISTANCE_REASON_CANCELLATION
-              "TIME" -> TIME_REASON_CANCELLATION
+              // "TIME" -> TIME_REASON_CANCELLATION
               "CANCELLED" -> CANCELLED_REASON_CANCELLATION
               "CLOSED" -> CLOSED_APP_REASON_CANCELLATION
               else -> DEFAULT_REASON_CANCELLATION
             }
-
-        MeetingRequestManager.meetingRequestViewModel?.removeFromMeetingRequestSent(senderUid) {
-          MeetingRequestManager.meetingRequestViewModel?.removeFromMeetingRequestInbox(senderUid) {
-            MeetingRequestManager.meetingRequestViewModel?.removeChosenLocalisation(senderUid) {
-              MeetingRequestManager.meetingRequestViewModel?.updateInboxOfMessages {}
-            }
+        if (stringReason != DEFAULT_REASON_CANCELLATION) {
+          MeetingRequestManager.meetingRequestViewModel?.removeFromMeetingRequestSent(senderUid) {
+            MeetingRequestManager.meetingRequestViewModel?.removeFromMeetingRequestInbox(
+                senderUid) {
+                  MeetingRequestManager.meetingRequestViewModel?.removeChosenLocalisation(
+                      senderUid) {
+                        MeetingRequestManager.meetingRequestViewModel?.updateInboxOfMessages {}
+                      }
+                }
           }
+          MeetingRequestManager.meetingRequestViewModel?.stopMeetingRequestTimer(senderUid, this)
+          showNotification("Cancelled meeting with $senderName", stringReason)
```

Comment:
Hardcoded string?

Reply:
Yes, the problem is that we have the variable senderName in the string, I could add a constant for "Cancelled meeting with" and then concatenate the two strings, but i felt that this way it is cleaner, what do you think ?

Reply:
I'm ambivalent. At the same time I understand you're rational but still it's a String that the users see that fits in strings.xml. Do as you think is best.